server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryFileCreator.java

```diff
@@ -27,47 +43,55 @@ public void createFiles() throws IOException, JSONException {
         createManifest();
         createBuildpackAdditionsFile();
         createDeployScript();
+        insertFiles();
     }
 
     private void createManifest() throws IOException {
-        String manifestContent;
-        manifestContent = createManifestHead();
-        manifestContent += createEnvironmentVariables();
-        manifestContent += createService();
-        fileAccess.access("/manifest.yml").append(manifestContent).close();
+        createManifestHead();
+        createAttributes();
+        createEnvironmentVariables();
+        createService();
     }
 
-    private String createManifestHead() {
-        return "applications: \n- name: " + app.getAppName();
+    private void createManifestHead() throws IOException {
+        String manifestHead = MANIFESTHEAD + "- " + NAMEBLOCK + ": " + app.getAppName();
```

Comment:
You could rewrite it like this (would enhance readability):
`String manifest = String.format("%s- %s: %s", MANIFESTHEAD, NAMEBLOCK, app.getAppName())`
This comment applies to all strings you build that way